tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/util/ProjectUtil.java

```
@@ -609,9 +609,6 @@ else if (projectRecord.getProjectName().endsWith(ISDKConstants.WEB_PLUGIN_PROJEC
 			pluginType = guessPluginType(fpwc);
 		}
 
-		SDKPluginFacetUtil.configureProjectAsSDKProject(
```

I don't think we could remove it? why do you remove this code?

---

Changed, i should ignore this error instead of removing it.